client/context_test.go

```diff
@@ -1,4 +1,4 @@
-package context_test
+package client_test
```

Comment:
revert this?

Reply:
We can't use multiple package names in a directory, and I feel tests in client are better under client_test